omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -35,6 +35,10 @@
             # load isn't supported by windows
             delete "#{conf_dir}/load.d"
 
+            # remove unused configs
+            delete "#{conf_dir}/process.d"
```

Comment:
you should not remove this dir, it's actually for the process _check_, not the process agent. I double checked and we already removed the process-agent yaml config in an ealier PR.